apps/showcase/src/app/app-routing.module.ts

```diff
@@ -12,6 +12,7 @@ const appRoutes: Routes = [
   {path: 'home', loadComponent: () => import('./home/index').then((m) => m.HomeComponent), title: 'Otter Showcase - Home'},
   {path: 'run-app-locally', loadComponent: () => import('./run-app-locally/index').then((m) => m.RunAppLocallyComponent), title: 'Otter Showcase - Run App Locally'},
   {path: 'sdk', loadComponent: () => import('./sdk/index').then((m) => m.SdkComponent), title: 'Otter Showcase - SDK'},
+  {path: 'sdk-intro', loadComponent: () => import('./sdk-intro/index').then((m) => m.SdkIntroComponent), title: 'Otter Showcase - SDK Introduction'},
```

Comment:
It would be nice to add this page to the e2e tests to avoid regressions (https://github.com/AmadeusITGroup/otter/tree/main/apps/showcase/e2e-playwright/sanity)

Reply:
I will do this in a separate PR since all of the screenshots will be updated with both Introduction and SDK Training in the sidenav